RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -107,12 +107,15 @@ private void _configService_SettingsChanged(object sender, ConfigurationChangedE
         }
 
         private ObservableCollection<IInspectionResult> _results = new ObservableCollection<IInspectionResult>();
-        public ObservableCollection<IInspectionResult> Results
+        public IEnumerable<IInspectionResult> Results
```

Comment:
why did you change this part? ... Also: Why did you change this file? #3485 is about the sorting in the Settings, not in the Results View...